### PR TITLE
docs(label): Adds React documenatation for Label component

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

**Comment by diegonvs** [comment]
Depends on #2063

File: next.clayui.com/content/docs/components/label.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+packageNpm: "@clayui/label"
+packageVersion: "3.0.0"
+packageStatus: "Beta"
+---
+
+import {Label, LabelClosingActions, LabelDisplayTypes, LabelLink, LabelSizes, LabelWithChildren} from '../../../src/components/clay/Label';
+
+<div class="nav-toc">
+
+- [Variants and Best Pratices](#variants-and-best-pratices)
```
Comment (matuzalemsteles) [inline next.clayui.com/content/docs/components/label.mdx]: I would say the same thing here that I mentioned in the button documentation. https://github.com/liferay/clay/pull/2062#discussion_r294529739

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by matuzalemsteles** [comment]
Resent #2089

**Pull request closed without merging**